cpp/dolfinx/fem/Form.h

```diff
@@ -93,12 +93,11 @@ class Form
   Form(const std::vector<std::shared_ptr<const FunctionSpace>>& function_spaces,
        const std::map<
            IntegralType,
-           std::pair<
-               std::vector<std::pair<
-                   int, std::function<void(T*, const T*, const T*,
-                                           const scalar_value_type_t*,
-                                           const int*, const std::uint8_t*)>>>,
-               const mesh::MeshTags<int>*>>& integrals,
+           std::map<int, std::pair<std::function<
+                                       void(T*, const T*, const T*,
+                                            const scalar_value_type_t*,
+                                            const int*, const std::uint8_t*)>,
+                                   std::vector<std::int32_t>>>>& integrals,
```

Comment:
Is `std::vector` the best container, it could it be a span? (or possibly a C++20 range)